.github/ISSUE_TEMPLATE/bug_report.md

```
@@ -7,12 +7,7 @@ assignees: RDXFGXY1
 
 ---
 
----
-name: 🐞 Bug Report
-about: Report an issue or unexpected behavior with the NullTracker Discord Bot
-title: "[Bug] "
-labels: bug
-assignees: ''
+#Bug report
```

markdownlint-cli2 (0.18.1)

No space after hash on atx style heading

(MD018, no-missing-space-atx)